UPGRADE.md

```diff
@@ -20,21 +20,20 @@ Finally, the caching mechanism is enabled by default.
 CLI options
 -----------
 
-1.x           | 2.0           | Description                                     | Note
-------------- | ------------- | ----------------------------------------------- | ----
-              | --allow-risky | Are risky fixers allowed                        |
-              | --cache-file  | The path to the cache file                      | option was added
---config      |               | Config class codename                           | option was removed
---config-file | --config      | The path to a .php_cs file                      | option was renamed
---diff        | --diff        | Show diff                                       |
---dry-run     | --dry-run     | Run in dry-run mode                             |
---fixers      |               | Coding standard fixers                          | option was removed, see --rules
---format      | --format      | Choose format                                   |
---level       |               | Coding standard level                           | option was removed, see --rules
-              | --path-mode   | Should the finder from configuration be         | option was added
-              |               | overriden or intersected with `path` argument   |
-              | --rules       | Rules to be used                                | option was added
-              | --using-cache | Does cache should be used                       | option was added
+1.x           | 2.0           | Description                                            | Note
+------------- | ------------- | ------------------------------------------------------ | ----
+              | --allow-risky | Are risky fixers allowed                               |
+              | --cache-file  | The path to the cache file                             | option was added
+--config      |               | Config class codename                                  | option was removed
+--config-file | --config      | The path to a .php_cs file                             | option was renamed
+--diff        | --diff        | Show diff                                              |
+--dry-run     | --dry-run     | Run in dry-run mode                                    |
+--fixers      |               | Coding standard fixers                                 | option was removed, see --rules
+--format      | --format      | Choose format                                          |
+--level       |               | Coding standard level                                  | option was removed, see --rules
+              | --path-mode   | Should `path` argument override the finder from config | option was added
```

Comment:
Like the idea of not taking two rows for one line of text (two lines would be ok).
The intersect part got lost (and is an important change) in the new text, I think this should be added again.